### ISSUE TITLE
Develop breaks if function with one argument as type annotation

### ISSUE THREAD

**Issue by Benjamin-Lee** [NiklasRosenstein/pydoc-markdown#92]
If your file is `testy_mctestyface.py`:

```python
def foo(bar: int) -> int:
    """Docstring goes here."""
    return bar + 1
```

The following error is generated:
```
~/D/P/R/hypothesis-bio ❯❯❯ pydoc-markdown --modules testy_mctestyface --search-path .
Traceback (most recent call last):
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/bin/pydoc-markdown", line 10, in <module>
    sys.exit(_entry_point())
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/main.py", line 122, in <lambda>
    _entry_point = lambda: sys.exit(main())
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/main.py", line 117, in main
    pydocmd.load_module_graph()
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/main.py", line 74, in load_module_graph
    loader.load(loader.config, self.graph)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 512, in load
    module = parse_file(None, filename, module_name)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 77, in parse_file
    return Parser().parse(parse_to_ast(code, filename), filename, module_name)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 101, in parse
    self.parse_declaration(module, node)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 115, in parse_declaration
    return self.parse_funcdef(parent, node, False, decorators)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 170, in parse_funcdef
    args = self.parse_parameters(parameters)
  File "/Users/BenjaminLee/.virtualenvs/hypothesis_bio/lib/python3.7/site-packages/pydoc_markdown/contrib/loaders/python.py", line 219, in parse_parameters
    result.append(Argument(parameters.children[1].value, None, None, Argument.POS))
AttributeError: 'Node' object has no attribute 'value'
```

Removing the `int` annotation for the argument results in a working markdown output:

````
# Table of Contents

  * [testy\_mctestyface](#testy_mctestyface)
    * [foo](#testy_mctestyface.foo)

# `testy_mctestyface`


## `foo()`

```python
def foo(bar)
```

Docstring goes here.
````

**Comment by Benjamin-Lee** [comment]
Also, note that the output type annotation is being ignored.

**Comment by NiklasRosenstein** [comment]
Hey @Benjamin-Lee , thanks for reporting this.

We've had this error before in #89, where it was supposed to be fixed. But it appears that a single argument is a special case that does not get wrapped in a `typedargslist` node, which results in the error you are seeing. 😢 

I've fixed it in b2064d3
